.github/workflows/ci.yml

```diff
@@ -0,0 +1,85 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
```suggestion
    branches: master
```

Reply:
Does this work? I though this part is required to be an array. I couldn't find an example with a string [in the docs](https://docs.github.com/actions/reference/events-that-trigger-workflows#configuring-workflow-events)... Do you have a working example?

Reply:
I have many working examples, but they are all private repos. Anyways this is /nit